primes.rb

```diff
@@ -1,6 +1,15 @@
 class Primes
   def self.sum_to(limit = 100)
     # TODO - add your prime number solution here...
-    "I'm working on it!"
+    require 'prime'
+
+p = Prime.new
+(1..100).each do |n|
+  q = p.next()
+  puts "#{n}: #{q}" 
+end
+
+Prime.each(100).reduce(:+)
```

Comment:
Probably doesn't show me the extent of your Ruby skills; have you looked at some of the Prime generating algorithms?  https://en.wikipedia.org/wiki/Sieve_of_Eratosthenes